library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/GroupResourcePack.java

```diff
@@ -131,9 +137,11 @@ public void listResources(ResourceType type, String namespace, String startingPa
 			ResourcePack.ResourceConsumer consumer) {
 		var packs = this.namespacedPacks.get(namespace);
 
-		// Iterating backwards as higher-priority packs are placed at the end.
-		for (var pack : packs) {
-			pack.listResources(type, namespace, startingPath, consumer);
+		if (packs != null) {
+			// Iterating backwards as higher-priority packs are placed at the end.
+			for (var pack : packs) {
```

Comment:
Is this comment actually relevant ? It seems copy-pasted from above

Reply:
It's the same `for` statement, just moved under the `null` check, so the comment is still relevant.